test/Helper/Navigation/AbstractTest.php

```diff
@@ -89,10 +86,6 @@ protected function setUp(): void
     {
         $cwd = __DIR__;
 
-        $this->routeMatchType = class_exists(V2RouteMatch::class)
```

Comment:
This conditional indicates that we can deal with V2 **or** V3. Can we somehow reduce that via `composer.json` restrictions (support only one version)?

Reply:
I've made the minimum version of Laminas\Navigation ^2.8.1 in dev which is the version where support was added for Laminas\Router vs Laminas\Mvc\Router.

Reply:
Does `"laminas/laminas-router": "^3.0.1",` cover that then?

Reply:
I assumed that if --dev will only install v3, then it's simply not possible to test against v2 without changing the dev requirement of router to 2||3 or something like that - so 2.8.1 of navigation fixes the test failures and renders the checks for v2 router obsolete???

Reply:
P.S. I think that the dev requirement of navigation could actually be bumped 2.9.x due to 7.3 support getting added around that release

Reply:
We can add a `"conflict": {"laminas/laminas-router": "<3.0.1"}` to solve that